counsel.el

```diff
@@ -6206,7 +6212,10 @@ The buffers are those opened during a session of `counsel-switch-buffer'."
            (message ""))))
       (t
        (with-ivy-window
-         (switch-to-buffer (ivy-state-buffer ivy-last)))))))
+         (if counsel-switch-buffer-display-empty-buffer-when-no-preview
+             (push (switch-to-buffer " *no preview*")
```

Comment:
Nit: perhaps something like `" *counsel no preview*"` would be safer and more informative?